pallets/block-reward/Cargo.toml

```diff
@@ -1,40 +1,42 @@
 [package]
 name = "pallet-block-reward"
-version = "2.0.0"
```

Comment:
We should use 2.1.0 or 2.0.1 instead of 3.0.0. The reason is we just improved some small functionalities there. Therefore, it's shouldn't a major change. The minor change is because we have the API break; however, because we haven't released it yet, it's also possible to use the build version change.

Reply:
Yeah, you are right... I think 2.1.0 is better than 3.0.0 because we renamed a dispatchable function, which is the main interface of this pallet... `set_hard_cap` to `set_max_currency_supply`